tests/wpt/mozilla/tests/mozilla/mediaElementAudioSource_closed_context_crash.html

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
```

Comment:
This file needs a `-crash.html` suffix (per https://web-platform-tests.org/writing-tests/crashtest.html#crashtest-tests) and we should move it to https://github.com/servo/servo/tree/main/tests/wpt/tests/webaudio/the-audio-api/the-mediaelementaudiosourcenode-interface